itest/lib/appStep/api/search.js

```diff
@@ -0,0 +1,74 @@
+/* @flow */
+
+import {Application} from "spectron"
+
+import logStep from "../util/logStep"
+import waitForResults from "../util//waitForResults"
+import {selectors} from "../../../../src/js/test/integration"
+
+export default async (app: Application, searchText: string) => {
```

Comment:
Mason and I recently agreed upon a convention where the default export is always at the bottom of the file, and private functions are on top. Since javascript has all these imports, it's sometimes hard to find the default export when it's in the middle of the file. It's not what I usually do in other languages, but I think it works well for javascript. Would you be willing to adopt that convention here too?

Reply:
Absolutely. I had thought to put it at the top as the first useful thing to be observed, but your import point makes sense.